camel/toolkits/hybrid_browser_toolkit/ws_wrapper.py

```
@@ -0,0 +1,487 @@
+                    # Check if this is the response we're waiting for
+                    if response.get('id') == message_id:
+                        if not response.get('success'):
+                            raise RuntimeError(
+                                f"Command failed: {response.get('error')}"
+                            )
+                        return response['result']
+
+                except asyncio.TimeoutError:
+                    raise RuntimeError(
+                        f"Timeout waiting for response to command: {command}"
+                    )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)